### PR TITLE
fix: min metric to hourly metric

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What this PR does / why we need it**:
Seems like p95 query latency dropped significantly after 3pm

The update ticker uses login.MetricsCollectionInterval which (based on comment) was suppose to be set to 4 hours, but it's set to 4 minutes

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by kalleep — APPROVED** [review]

**Comment by eleijonmarck** [comment]
@kalleep the tests that fail are from https://github.com/grafana/grafana/pull/55056

where the ordering of the results could potentially come back differently.

Here is a fix for that test https://github.com/grafana/grafana/pull/55232

File: pkg/services/login/model.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ type LoginStats struct {
 
 const (
 	ExporterName              = "grafana"
-	MetricsCollectionInterval = time.Second * 60 * 4 // every 4 hours, indication of duplicate users
+	MetricsCollectionInterval = time.Second * 60 * 60 * 4 // every 4 hours, indication of duplicate users
```
Comment (marefr) [inline pkg/services/login/model.go:17]: Nit. More readable?

```suggestion
	MetricsCollectionInterval = time.Hour * 4 // every 4 hours, indication of duplicate users
```

**Review by marefr — APPROVED** [review]
LGTM, but added a nit/suggestion

File: pkg/services/login/model.go (hunk 1 above)
Comment (eleijonmarck) [reply pkg/services/login/model.go:17]: more readable for sure!

**Review by Jguer — APPROVED** [review]

**Pull request merged**